### PR TITLE
Ray Dashboard IAP

### PR DESCRIPTION
**Pull request by blackzlq**
Integrate IAP with Ray Dashboard
Tested-by: zlq

### CODE REVIEW

File: applications/rag/frontend/main.tf [hunk 1]
```diff
@@ -80,9 +80,6 @@ resource "kubernetes_service" "rag_frontend_service" {
   metadata {
     name      = "rag-frontend"
     namespace = var.namespace
-    annotations = var.add_auth ? {
```
Comment (imreddy13) [inline applications/rag/frontend/main.tf:83]: Why did we delete this?

File: modules/iap/iap.tf [hunk 2]
```diff
@@ -108,6 +108,22 @@ resource "helm_release" "iap" {
   }
 }
 
+resource "kubernetes_annotations" "annotation" {
```
Comment (imreddy13) [inline modules/iap/iap.tf:111]: Did we resolve the issue with annotations + reapply?

**Review by imreddy13 — APPROVED** [review]

**Review by jindijamie — APPROVED** [review]

File: applications/rag/frontend/main.tf (hunk 1 above)
Comment (blackzlq) [reply applications/rag/frontend/main.tf:83]: Discussed with Umesh, we moved all annotation operation back to IAP

File: modules/iap/iap.tf (hunk 2 above)
Comment (blackzlq) [reply modules/iap/iap.tf:111]: No, Umesh is working on it

**Pull request merged**